device/iot-device-samples/file-upload-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/FileUploadSample.java

```diff
@@ -129,53 +129,40 @@ private static void uploadFileOrDirectoryRecursive(DeviceClient client, String b
 
     private static void uploadFile(DeviceClient client, String baseDirectory, String relativeFileName) throws IOException, URISyntaxException {
         File file = new File(baseDirectory, relativeFileName);
-        try (InputStream inputStream = new FileInputStream(file))
```

Comment:
The diff looks weird for some reason but line 132 and line 150 are the main things. We no longer need FileInputStream and we use the blobClient.uploadFromFile.